tests/.editorconfig

```diff
@@ -13,6 +13,7 @@ csharp_style_var_elsewhere = true:none
 # where we want to marshal the continuations back to it.
 dotnet_diagnostic.CA2007.severity = none
 dotnet_diagnostic.CA1822.severity = none
+dotnet_diagnostic.CA2016.severity = none        # ForwardCancellationTokenToInvocations
```

Comment:
Suggestion?

Reply:
It's all noise in the unit tests, hence turned it off just under it.

Reply:
Missed that this was in the tests file.